docs/openj9_signals.md

```diff
@@ -0,0 +1,174 @@
+| `SIGRTMIN (34)`   | Con  | Used by the VM for thread introspection                       | - |
+| `SIGRTMIN +1 (35)`| Con  | Used by the VM for Runtime Instrumentation (Linux for IBM Z systems only)| - |
+| `SIGRTMAX -2 (62)`| Con  | Used by the `java.net` class library code                                         | - |
+| `SIGCHLD (17)`    | Con  | Used by the `java.lang.Process` implementation                     | - |
+
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Notes:**
+
+- use of `SIGRTMIN` is configurable with the `-Xdump:suspendwith=<num>` option
+
+## Signals on macOS
+
+| Signal            | Type | Description                                                        | Option to disable signal  |
+| ------------------|------|--------------------------------------------------------------------|-----------------------|
+| `SIGBUS (10)`     | Exc  | Incorrect memory access (data misalignment)                        | `-Xrs` or `-Xrs:sync` |
+| `SIGSEV (11)`     | Exc  | Incorrect memory access (write to inaccessible area)               | `-Xrs` or `-Xrs:sync` |
+| `SIGILL (4)`      | Exc  | Illegal instruction (attempt to call unknown machine instruction)) | `-Xrs` or `-Xrs:sync` |
+| `SIGFPE (8)`      | Exc  | Floating point exception (divide by zero)                          | `-Xrs` or `-Xrs:sync` |
+| `SIGABRT (6)`     | Err  | Abnormal termination, raised by the VM when a VM fault is detected | `-Xrs` or `-Xrs:sync` |
+| `SIGINT (2)`      | Int  | Interactive attention (CTRL-C), VM exits normally                  | `-Xrs` |
+| `SIGTERM (15)`    | Int  | Termination request, VM exits normally                             | `-Xrs` |
+| `SIGHUP (1)`      | Int  | Hang up, VM exits normally                                         | `-Xrs` |
+| `SIGQUIT (3)`     | Con  | Quit signal from a terminal, which triggers a Java dump by default | `-Xrs` |
+| `SIGTRAP (5)`     | Con  | Used by the JIT                                                    | `-Xrs` or `-Xrs:sync` |
+| `SIGCHLD (20)`    | Con  | Used by the `java.lang.Process` implementation                     | - |
+| `SIGUSR1 (30)`    | Con  | Used by the VM for thread introspection                                       | - |
+| `SIGIO (23)`      | Con  | Used by the `java.net` class library code                                        | - |
+
+## Signals on Windows
+
+| Signal            | Type | Description                                                            | Option to disable signal   |
+| ------------------|------|------------------------------------------------------------------------|-----------------------|
+| `SIGINT (2)`      | Int  | Interactive attention (CTRL-C), VM exits normally                      | `-Xrs` |
+| `SIGTERM (15)`    | Int  | Termination request, VM exits normally                                 | `-Xrs` |
+| `SIGBREAK`        | Con  | A break signal from a terminal. By default, this triggers a Java dump  | `-Xrs` |
```

Comment:
```suggestion
| `SIGBREAK`        | Con  | A break signal from a terminal. By default, this triggers a Java dump.  | `-Xrs` |
```
Should all the statements in the `Description` column end with a fullstop (.)?

Reply:
Entries in a table don't need to, same for a bulleted list, but I did accept your suggestion for the numbered list.